tests/integration/Cake.Core/Scripting/DefineDirective.cake

```diff
@@ -64,23 +66,40 @@ Task("Cake.Core.Scripting.DefineDirective.Cake")
     Assert.True(cake);
 });
 
-#if NET5_0
+#if NET5_0_OR_GREATER
     Task("Cake.Core.Scripting.DefineDirective.C#9")
     .Does(() =>
 {
-    // givem
+    // given
     var csharpNine = new CSharpNine(true);
     Assert.True(csharpNine.IsNine);
 });
 
 public record CSharpNine(bool IsNine);
 #endif
 
+#if NET6_0_OR_GREATER
+    Task("Cake.Core.Scripting.DefineDirective.C#10")
+    .Does(() =>
+{
+    // given
+    const string world = "world";
+    const string hello = "Hello";
+    const strong helloWorld = $"{hello} {world}!";
```

Comment:
`strong`? 😅

Reply:
WIP,  not done for review 😎

need to update Roslyn too and greater never made it into .NET so it needs to be  || NET6_0